test/geometry/warp.jl

```diff
@@ -84,3 +84,27 @@ end
     @test_broken zy[1].H ≈ man[1]
     @test zy[1].H ≈ -man[1] # TODO: Remove this test and make the above @test_broken -> @test after resolving the sign thing
 end
+
+@testset "RotMatrix vs. Non-RotMatrix" begin
+    img = rand(RGB{Float32}, 10, 10)
+    tgt = rand(RGB{Float32}, 10, 10)
+
+    f(t) = begin
+        out = ImageTransformations.warp(img, t, axes(img), zero(eltype(img)))
+        loss = image_mse(out, tgt)
+        loss
+    end
+
+    tfm1 = recenter(RotMatrix(π/8), center(img)) # using RotMatrix
+    tfm2 = AffineMap(SMatrix{2, 2, Float64, 4}([cos(π/8) -sin(π/8); sin(π/8) cos(π/8)]), tfm1.translation)
+
+    zy1 = Zygote.gradient(f, tfm1)
```

Comment:
Oh brilliant, are the `warp!` rules working as expected now?

Reply:
Yup